arangod/RocksDBEngine/RocksDBCuckooIndexEstimator.h

```diff
@@ -762,6 +765,8 @@ class RocksDBCuckooIndexEstimator {
     // If we get here we had to remove one of the elements.
     // Let's increas the cuckoo counter
     _nrCuckood++;
+    // and let's decrease the total so we don't have to recalculate later
+    _nrTotal -= counter;
```

Comment:
Good catch lost this info before.
However we should protect against integer underflow. (i do not know if it can ever happen, just to be sure.)

Reply:
It should never happen ... but I'm okay with changing it if you think that's safer.

Reply:
I think you are right, it can never happen.